indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -96,8 +96,17 @@
                                 </td>
                                 {% for item in static_columns %}
                                     {% if item.id == 'state' %}
-                                        <td class="i-table" data-searchable="{{ abstract.state.title | lower }}">
-                                            {{- abstract.state.title -}}
+                                        <td class="i-table" data-searchable="{{ abstract.public_state.title | lower }}">
+                                            {% if abstract.public_state.name == 'under_review' %}
+                                                <div class="i-tag state-badge under-review">
+                                                    {{- abstract.reviews|length|string + " " + ngettext("review on", "reviews on", abstract.reviews|length) + " "  +
```

Comment:
I think this line is super ugly and needs to be improved quite a bit ;)

- Don't use `+` to build strings (you can just `}} {{` in a template to add that space)
- Don't use multiple gettext calls in the same sentence. That doesn't work well depending on the structure it has in another language
- There's {% trans %}{% pluralize %}{% endtrans %}`